kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -22,9 +22,6 @@ serverStartPolicy: IF_NEEDED
 # Cluster name
 clusterName: cluster-1
 
-# Number of managed servers to generate for the domain
-configuredManagedServerCount: 5
-
```

Comment:
since we are now using a wdt model with configured cluster with 3 servers. model is either wdt_mdoel_configured.yaml or wdt_model_restricted_jrf_configured.yaml.

Reply:
The sample were allowing users to use wlst offline scripts or providing wdt models, but those support were based on docker-images samples that we are trying to replace with WIT in this task. I may be wrong though.

Reply:
I assume there's a hybrid approach that can use WIT in combination with WLST off-line generated domains, but could be wrong.  We should discuss with Monica, Derek, and Ryan:
(a) Is it required that our basic samples continue to demonstrate creating domains using WLST off-line (in addition to WDT)?
(b) If so, how do we go about demonstrating WIT in WLST off-line case?

Reply:
Any updates on this discussion?

Reply:
@ddsharpe @mriccell